use-case-guides/workday-upcoming-birthdays-using-reporting/README.md

```diff
@@ -0,0 +1,160 @@
+
+**This use case is made up of:**
+
+✨ **Triggers:** Natural Language
+
+🤲 **Slots**: None
+
+🏃‍♂️ **Actions:** Retrieve data from workday report
+
+📜 **Response Data:** The data from a RaaS report
+
+📚 **Guidelines:** Select fields to render from RaaS report
+
+Based on the needs of this use case, we should build a [Static Query](https://developer.moveworks.com/creator-studio/resources/design-pattern?id=static-query)
+
+## API Design
+
+There’s only 1 API needed to build this use case.
+
+### API #1: Get RaaS Report
+
+Since we want to get a RaaS report, we only need to reference the report in our API call and get the results from the report to display within Creator Studio.
+
+# **Prerequisites**
+
+- A connection to Workday, [follow our guide.](https://developer.moveworks.com/creator-studio/resources/authentication-guide?id=workday)
+- A user login to Workday.
+
+# **Walkthrough**
+
+## **Step 1: Get the URL for the Report**
+
+In this step, we will get the URL for the report we want to run within Creator Studio. We are using a pre-made report as part of this example but you can call any report that already exists or that you create using these steps.
+
+We are going to use the *Alert - Birthdays for this week and next* report to get the upcoming birthdays for this example.
+
+1. Login to Workday and search for “All Custom Reports” in the search bar.
+    
+    ![Search for all custom reports in workday](images/Untitled.png)
+    
+2. Filter *Custom Report Name* by “Birthday”.
+3. Check the box next to “Alert - Birthdays for this week and next” and filter.
+    
+    ![choose the "Alert - birthdays for this week and next"](images/Untitled%201.png)
+    
+4. Click on the link for “Alert - Birthdays for this week and next”.
+    
+    ![shows the report from the search results](images/Untitled%202.png)
+    
+5. From the ellipse drop-down, go to *Web Service → View URLs.*
+    
+    ![the ellipse for the report to show custom actions](images/Untitled%203.png)
+    
+6. You will get a dialogue to sign in, press cancel and copy the URL after *[wd2-impl-services1.workday.com](http://wd2-impl-services1.workday.com)* 
+    
+    ![browser window showing url](images/Untitled%204.png)
+    
+    which should look something like this:
+    
+    <pre>/ccx/service/customreport2/[your_tenant]/[your user_name]/Alert_-_Birthdays_for_this_week_and_next</pre>
```

Comment:
We don't support the "pre" HTML Tag

Reply:
weirdest thing, had it as '''text''' but it kept throwing off the checker. Seems to be working now though